lib/glific/seeds/seeds_migration.ex

```diff
@@ -407,4 +407,21 @@ defmodule Glific.Seeds.SeedsMigration do
     |> select([c, _p], c.organization_id)
     |> Repo.all()
   end
+
+  @spec update_localized_language() :: :ok
+  defp update_localized_language do
+    Glific.Settings.Language
+    |> where([l], l.label in ["English", "Hindi"])
+    |> update([l], set: [localized: true])
+    |> Repo.update_all([], skip_organization_id: true)
```

Comment:
Please check if we need to skip for global schema as well. 

Let's use shortcodes instead of labels.